agnoster.zsh-theme

```diff
@@ -122,10 +122,20 @@ prompt_status() {
 
 # Display current virtual environment
 prompt_virtualenv() {
+  local env='';
+
   if [[ -n $VIRTUAL_ENV ]]; then
+    env=$VIRTUAL_ENV
+  fi
+
+  if [[ -n $CONDA_DEFAULT_ENV ]]; then
```

Comment:
why not check this first and then use `elif` to explicitly fall back? This will make sure that this is the first choice if changes are made in the future.